roles/kubernetes/preinstall/handlers/main.yml

```diff
@@ -95,6 +95,10 @@
 
 - name: Preinstall | restart kube-apiserver crio/containerd
   shell: "{{ bin_dir }}/crictl pods --name kube-apiserver* -q | xargs -I% --no-run-if-empty bash -c '{{ bin_dir }}/crictl stopp % && {{ bin_dir }}/crictl rmp %'"
+  register: preinstall_remove_apiserver_container
```

Comment:
Why not `restart` instead of `remove`?

Reply:
I honestly didn't pay attention to that. Initially I was wondering why there is no retry part in `preinstall` section, while the same thing is present in `control-plane` section [here](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/control-plane/handlers/main.yml#L54-L60), so I just blindly copied it. I also noticed that here the task is called `restart kube-apiserver`, but in link above the same task is called `Remove apiserver container`, not exactly sure why though. Anyway, I changed the var name to `preinstall_restart_apiserver`. If you have any suggestions let me know 😎

Reply:
Hehe, that was what I guess.
I just didn't want to see another pull request which renames the variable name as a typo.
Current one seems good for me, thank you so much for updating.